package.json

```diff
@@ -16,6 +16,12 @@
         "lodash": "^4.17.13",
         "resolve-url-loader": "^3.1.0",
         "sass": "^1.15.2",
-        "sass-loader": "^8.0.0"
+        "sass-loader": "^8.0.0",
+        "vue-template-compiler": "^2.6.11"
```

Comment:
is this compiler needed?

Reply:
Yeah, mix unfortunately always requires this by default for some reason. It'll auto-add it as soon as you try to compile the assets.